### ISSUE TITLE
Page Creation: Install "debug" instead of profiler

### ISSUE THREAD

**Issue by weaverryan** [symfony/symfony-docs#8916]
In this intro page, we tell them to install the profiler: https://symfony.com/doc/current/page_creation.html#the-web-debug-toolbar-debugging-dream

I think it would be better to have them install `debug` (the debug pack: https://github.com/symfony/debug-pack/blob/master/composer.json), as it contains other useful things, like `var-dumper`.

Before doing this blindly, we need to try it and make sure there are no surprises or "too much" extra baggage :).

**Comment by javiereguiluz** [comment]
I agree ... and maybe it's easier to understand for readers too. Installing "debug" feels more generic about adding tools to help you while developing the app. Installing "profiler" is just like installing one specific tool.

**Comment by ArlingtonHouse** [comment]
Symfony 4 is a real struggle just to get basic things like the toolbar and profiler turned on.  The profiler composer command isn't enough, there are environment variables floating around like APP_DEBUG=1 and settings in the web_profiler.yaml file .

**Comment by ArlingtonHouse** [comment]
Related the page https://symfony.com/doc/current/components/debug.html is unclear-- where should the `Debug::enable(); ` be placed?  Controller? Also how does this interact with env settings?

**Comment by weaverryan** [comment]
Well, you’re reading the component documentation about the Debug component, which is only relevant if you’re *not* using the framework. But perhaps that’s not very clear :/

**Comment by weaverryan** [comment]
And actually, the composer command IS enough - the recipe should create the config files you need and the APP_ENV is set to 1 by default. Just wanted to clear that up for anyone else reading :)
